xbmc/rendering/RenderSystem.cpp

```diff
@@ -90,10 +90,7 @@ void CRenderSystemBase::ShowSplash(const std::string& message)
 
   if (!m_splashImage)
   {
-    std::string splashImage = "special://home/media/Splash.png";
-    if (!XFILE::CFile::Exists(splashImage))
-      splashImage = "special://xbmc/media/Splash.png";
-
+    std::string splashImage = CUtil::GetSplashPath();
     m_splashImage = std::unique_ptr<CGUIImage>(new CGUIImage(0, 0, 0, 0, g_graphicsContext.GetWidth(),
                                                        g_graphicsContext.GetHeight(), CTextureInfo(splashImage)));
```

Comment:
I would eliminate `splashImage` variable => `CTextureInfo(CUtil::GetSplashPath())`

Or make the var at least `const`.

Reply:
sure, why not